homeassistant/components/lutron.py

```diff
@@ -0,0 +1,81 @@
+"""
+Component for interacting with a Lutron RadioRA 2 system.
+
+Uses pylutron (http://github.com/thecynic/pylutron).
+"""
+
+import logging
+
+from homeassistant.components import group
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/thecynic/pylutron/archive/v0.1.0.zip#pylutron==0.1.0']
+
+DOMAIN = "lutron"
+
+# List of component names (string) your component depends upon.
+DEPENDENCIES = ['group', 'light']
+
+_LOGGER = logging.getLogger(__name__)
+
+LUTRON_CONTROLLER = None
+
+LUTRON_DEVICES = {'light': []}
+LUTRON_GROUPS = {}
+
+
+def setup(hass, base_config):
+    """Setup our skeleton component."""
+    global LUTRON_CONTROLLER
+
+    from pylutron import Lutron
+
+    config = base_config.get(DOMAIN)
+    LUTRON_CONTROLLER = Lutron(
+        config['lutron_host'],
+        config['lutron_user'],
+        config['lutron_password']
+    )
```

Comment:
There is no need for the `lutron_` prefix. We have constants for those variables and please use `username` instead of `user`.

Also, use `voluptuous` to ensure that the user is providing the needed configuration in the form you need it. See other components or the [docs](https://home-assistant.io/developers/development_validation/) for details.